docs/Homebrew-on-Linux.md

```diff
@@ -45,6 +45,9 @@ brew install hello
 
 If you're using an older distribution of Linux, installing your first package will also install a recent version of *glibc* and *gcc*. Use `brew doctor` to troubleshoot common issues.
 
+> [!NOTICE]
+> Please note that unlike macOS, Homebrew does not use a sandbox when building on Linux, so formulae may install outside the Homebrew prefix.
```

Comment:
`may not only read but also write arbitrary files anywhere outside the Homebrew prefix`

Thoughts?

Reply:
I would turn them around, since I'd be more worried about reads than writes. But clarification is good either way

Reply:
Our macOS sandbox doesn't block reads either.